### ISSUE TITLE
This includes a Softmax within the loss function

### ISSUE THREAD

**Issue by wp99cp** [wp99cp/bachelor-thesis#1]
Thus we should remove our softmax form the model?

https://github.com/wp99cp/bachelor-thesis/blob/09fbe5f0403047d707df9d31446414d274f86ba5/models/unet/model/training.py#L42
